framework/web/ErrorAction.php

```diff
@@ -98,6 +98,13 @@ public function init()
      */
     public function run()
     {
+        $response = Yii::$app->getResponse();
+        if ($this->exception instanceof HttpException) {
```

Comment:
Because in case of ErrorAction, there's a way to trigger it w/ exception without actually going through error handler.